apps/cpblscores/cpbl_scores.star

```diff
@@ -0,0 +1,455 @@
+]
+
+databaseTypeOptions = [
+    schema.Option(
+        display = "Past 5 games & feature 5 games",
+        value = "both",
+    ),
+    schema.Option(
+        display = "Past 5 games",
+        value = "past",
+    ),
+    schema.Option(
+        display = "Feature 5 games for specify team or 10 League games",
+        value = "feature",
+    ),
+]
+
+displayTopOptions = [
+    schema.Option(
+        display = "League Name",
+        value = "league",
+    ),
+    schema.Option(
+        display = "Clock & Game Info",
+        value = "time",
+    ),
+    schema.Option(
+        display = "Game Info Only",
+        value = "gameinfo",
+    ),
+]
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Location(
+                id = "location",
+                name = "Location",
+                desc = "Location for which to display time.",
+                icon = "locationDot",
+            ),
+            schema.Dropdown(
+                id = "selectedTeam",
+                name = "Team Focus",
+                desc = "Only show games for selected team.",
+                icon = "baseballBatBall",
+                default = teamOptions[1].value,
+                options = teamOptions,
+            ),
+            schema.Dropdown(
+                id = "rotationSpeed",
+                name = "Rotation Speed",
+                desc = "Amount of seconds each score is displayed.",
+                icon = "gauge",
+                default = rotationOptions[1].value,
+                options = rotationOptions,
+            ),
+            schema.Dropdown(
+                id = "databaseType",
+                name = "Game Info",
+                desc = "Select past or feature game info you want to show.",
+                icon = "calendar",
+                default = "past",
+                options = databaseTypeOptions,
+            ),
+            schema.Dropdown(
+                id = "displayTop",
+                name = "Top Display",
+                desc = "A toggle of what to display on the top shelf.",
+                icon = "clock",
+                default = displayTopOptions[1].value,
+                options = displayTopOptions,
+            ),
+            schema.Color(
+                id = "displayTimeColor",
+                name = "Top Display Color",
+                desc = "Select which color you want the top display to be.",
+                icon = "brush",
+                default = "#ffff00",
+                palette = [
+                    "#7AB0FF",
+                    "#BFEDC4",
+                    "#78DECC",
+                    "#DBB5FF",
+                ],
+            ),
+            schema.Text(
+                id = "apikey",
+                name = "API Key",
+                desc = "Enter your theSportDB API key",
+                icon = "key",
+            ),
+        ],
+    )
+
+def get_cachable_data(url, ttl_seconds = CACHE_TTL_SECONDS):
```

Comment:
The http module now has built-in cache functionality. Just pass `tll_seconds` to `http.get()`. Would be great if you could switch over to using that!

Reply:
Hey @matslina  - Thanks for the advice. I committed to adopting built-in cache functionality in http module.